assets/src/web-stories-block/block/constants.js

```diff
@@ -95,6 +95,38 @@ export const VIEW_TYPES = [
     icon: CAROUSEL_VIEW_TYPE_ICON,
   },
 ];
+
+/**
+ * Stories 'order-by' values.
+ */
+export const ORDER_BY_OPTIONS = {
+  date: {
```

Comment:
Maybe `new-to-old` for consistency? Although I do like the simplicity of `date`...

Reply:
I had kept it for simplicity but guess consistency is also important for the maintainability of the source. I'll update it.